frontend/ui/widget/container/widgetcontainer.lua

```diff
@@ -85,8 +94,8 @@ end
 
 --[[--
 WidgetContainer will pass event to its children by calling their handleEvent
-methods. If no child responded to the event (by returning true), it will call its own
-handleEvent method.
+methods. If no child responded to the event (by returning true), it will try
+consume the event by itself.
```

Comment:
to consume

Is consume the right word though? Is it trying to act on the event or is it trying to stop propagation (=consume)?